src/app/components/planner/planner.component.scss

```diff
@@ -278,6 +278,12 @@
 						.sidebar-coming-up {
 							margin-bottom: 0.5rem;
 						}
+						.btn-group{
```

Comment:
Like what you were saying in the PR description, I think it would semantically make more sense to use more specific class names. If the buttons don't have a specific class name, then maybe also assigning them appropriate classes.

Reply:
Also, formatting issue: to remain consistent with the other SCSS, there should be a space after the class name and before the curly bracket